app/assets/javascripts/pretty-text/engines/discourse-markdown/watched-words.js

```diff
@@ -107,7 +107,9 @@ export function setup(helper) {
               currentToken.type === "span_open") &&
             currentToken.attrs &&
             currentToken.attrs.some(
-              (attr) => attr[0] === "class" && attr[1] === "hashtag"
+              (attr) =>
+                attr[0] === "class" &&
+                (attr[1] === "hashtag" || attr[1].includes("hashtag-cooked"))
```

Comment:
maybe just `attr[1].includes("hashtag")` ? the use of includes vs === is confusing me a bit here?

Reply:
Yeah good point it will work for the old way there too :+1:

Reply:
oh it is class names, so yeah includes hashtag is not perfect, but probably fine and a bit simpler.